WooCommerce/Classes/AppDelegate.swift

```diff
@@ -49,6 +49,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     ///
     private lazy var appleIDCredentialChecker = AppleIDCredentialChecker()
 
+    private let universalLinkRouter = ServiceLocator.universalLinkRouter
```

Comment:
Do we need this to exist in `ServiceLocator` can't it just exist within the app delegate?

Reply:
I wanted to encapsulate the initialization call with the default routes, but it is true that we do not need the `ServiceLocator` for that, we can do it directly with a static method in `UniversalLinkRouter`, called in the `AppDelegate`. Thanks for the tip! Changed in 3ef6382ed74537dac3780a644296316ce71432c6